Xamarin.Forms.Platform.iOS/CollectionView/HorizontalDefaultCell.cs

```diff
@@ -3,13 +3,12 @@
 
 namespace Xamarin.Forms.Platform.iOS
 {
-	// TODO hartez 2018/09/12 21:43:54 The "List" part of all these cell names needs to go	
-	internal sealed class DefaultHorizontalListCell : DefaultCell
+	internal sealed class HorizontalDefaultCell : DefaultCell
 	{
 		public static NSString ReuseId = new NSString("Xamarin.Forms.Platform.iOS.DefaultHorizontalListCell");
```

Comment:
I think this value is wrong here with "horizontal" in the name... But this might be another issue.